src/Entities/ProductEntity.php

```diff
@@ -1,7 +1,10 @@
 <?php
 
+
 namespace App\Entities;
 
+
+use App\Validators\NameValidator;
```

Comment:
These changes were previously removed, not sure why they've been added again, I reckon just delete all changes from this file